code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1768,3 +1768,10 @@
 	color = "#020202"
 	taste_description = "bananas"
 	can_synth = TRUE
+
+/datum/reagent/adhesive
+	name = "Adhesive"
+	description = "A Sticky substance that used to be made out of horses but is now creatable with chemicals"
```

Comment:
description = "A sticky substance that used to be made out of horses, but is now created with chemicals."

Reply:
@Ktlwjec 

![image](https://user-images.githubusercontent.com/24573384/83137301-3974c700-a0e9-11ea-9d64-acb183c7cfcf.png)